test/views/pages/post/post_page_values_test.dart

```diff
@@ -0,0 +1,107 @@
+import "package:firebase_core/firebase_core.dart";
+import "package:flutter/material.dart";
+import "package:flutter_test/flutter_test.dart";
+import "package:proxima/models/database/post/post_id_firestore.dart";
+import "package:proxima/models/ui/post_overview.dart";
+import "package:proxima/views/pages/post/post_page_widget/complete_post_widget.dart";
+
+import "../../../mocks/data/post_overview.dart";
+import "../../../mocks/providers/provider_post_page.dart";
+import "../../../mocks/services/setup_firebase_mocks.dart";
+
+void main() {
+  // Custom post for testing specific date and distances
```

Comment:
Even if it's easier to understand the test with the custom post declared here, isn't it better if this mock data is declared in the file`test/mocks/data/post_overview.dart`

Reply:
I don't think this post data will be useful to anyone else as it is closely coupled with the current test and I placed it here to avoid polluting the global testing namespace.

That being said, to stay consistent with the other mock data I applied your suggestion in 144bea8